test/clj/puppetlabs/trapperkeeper/services/webserver/jetty9_service_test.clj

```diff
@@ -84,38 +96,28 @@
    (validate-ring-handler base-url config http-get-options :default))
   ([base-url config http-get-options server-id]
     (with-app-with-config app
-      [jetty9-service]
+      [jetty9-service
+       hello-webservice]
       config
-      (let [s                   (tk-app/get-service app :WebserverService)
-            add-ring-handler    (partial add-ring-handler s)
-            body                "Hi World"
-            path                "/hi_world"
-            ring-handler        (fn [req] {:status 200 :body body})]
-        (add-ring-handler ring-handler path {:server-id server-id})
-        (let [response (http-get
-                         (format "%s%s/" base-url path)
-                         http-get-options)]
-          (is (= (:status response) 200))
-          (is (= (:body response) body)))))))
+      (let [response (http-get
+                      (format "%s%s/" base-url hello-path)
+                      http-get-options)]
+        (is (= (:status response) 200))
```

Comment:
I guess in our next round of "make the tests less stupid" we should pull these assertions directly into the tests rather than doing them in a helper function?  Don't think we need to worry about that for this PR...